From 2c92c4edab4bc1d5370e24c44be27943141e9b37 Mon Sep 17 00:00:00 2001
From: arun <arun.radhakrishnan@intel.com>
Date: Thu, 24 Mar 2016 18:00:30 -0600
Subject: [PATCH 15/15] iio: Adding support hrtimer support for iio

The CHT_MRD Magnetometer(ak09911) does not have an interrupt pin
connected and the polling method does not seem to achieve the
desired data read frequency. Enabling hrtimer to improve the
frequency characteristics.

Change-Id: I33286270bd8890b2d624a08f8451013dd44eee85
Signed-off-by: Arun Radhakrishnan <arun.radhakrishnan@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-27736
Orig-Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26476
Reviewed-on: https://android.intel.com:443/490334
---
 arch/x86/configs/x86_64_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index a4ffde1..ed720b6 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -3506,6 +3506,9 @@ CONFIG_STAGING=y
 #
 # Triggers - standalone
 #
+CONFIG_IIO_CONFIGFS=y
+CONFIG_IIO_HRTIMER_TRIGGER=y
+CONFIG_IIO_SW_TRIGGER=y
 # CONFIG_IIO_PERIODIC_RTC_TRIGGER is not set
 # CONFIG_IIO_SIMPLE_DUMMY is not set
 # CONFIG_FB_SM7XX is not set
-- 
1.9.1

